vm/class.go

```diff
@@ -99,7 +99,7 @@ func builtinClassCommonClassMethods() []*BuiltinMethodObject {
 				class, ok := receiver.(*RClass)
 
 				if !ok {
-					return t.vm.InitUnsupportedMethodError(sourceLine, "#new", receiver)
+					return t.vm.InitNoMethodError(sourceLine, "new", receiver)
```

Comment:
I think we can still use `InitNoMethodError` here (and below)?

Reply:
I'd check around this

Reply:
Applied line 159 and some more.